developerGuide.t2t

```diff
@@ -374,26 +369,73 @@ When NVDA receives input, it looks for a matching gesture binding in a particula
-You can also set the "scriptCategory" attribute on the plugin class, which will be used for scripts which do not specify a category.
+The category can be set for individual scripts, but you can also set the "scriptCategory" attribute on the plugin class, which will be used for scripts which do not specify a category.
 There are constants for common categories prefixed with SCRCAT_ in the inputCore and globalCommands modules.
 The script will be listed under the specified category in the Input Gestures dialog.
 If no category is specified, the script will be categorized under "Miscellaneous".
 
++++ Defining script properties +++
+For NVDA 2018.3 and above, the recommended way to set script properties is by means of the so called script decorator.
+In short, a decorator is a function that modifies the behavior of a particular function.
+The script decorator modifies the script in such a way that it will be properly bound to the desired gestures.
+Furthermore, it ensures that the script is listed  with the description you specify, and that it is categorised under the desired category in the input gestures dialog.
+
+In order for you to use the script decorator, you will have to import it from the scriptHandler module.
+```
+from scriptHandler import script
+```
+
+After that, just above your script definition, add the script decorator, providing it the desired arguments:
+```
+--- start ---
+	@script(
+		description=_("Speaks the date and time"),
+		category=inputCore.SCRCAT_MISC,
+		gestures=["kb:NVDA+shift+t", "kb:NVDA+alt+r"]
+	)
+	def script_sayDateTime(self, gesture):
+
+--- end ---
+```
+
+After this, your script will be listed in the input gestures dialog under the "Miscellaneous" category.
+It will have the description "Speaks the date and time", and will be bound to the "NVDA+shift+t" and "NVDA+alt+r" key combinations on the keyboard.
+
+The following keyword arguments can be used when applying the script decorator:
+- description: A short, translatable description of the script to be used in the input gestures dialog.
+- category: The category of the script displayed in the input gestures dialog.
+- gesture: A single gesture associated with this script, e.g. "kb:NVDA+shift+r".
+- gestures: A list of multiple gestures associated with this script, e.g. ["kb:NVDA+shift+r", "kb:NVDA+alt+t"].
+  When both gesture and gestures are specified, they add up together.
+- canPropagate: Whether this script should also apply when it belongs to a  focus ancestor object.
+  For example, this can be used when you want to specify a script on a particular foreground object, or another object in the focus ancestry which is not the current focus object.
+  This option defaults to False.
+- bypassInputHelp: Whether this script should run when input help is active.
+  This option defaults to False.
+-
+
+Though  the script decorator makes the script definition process a lot more easier, there are more ways of binding gestures and setting script properties.

```

Comment:
There is a double space after 'Though'. I would also suggest removing the word 'more', "a lot easier" will suffice.